examples/http-tinygo-outbound-http/spin.toml

```diff
@@ -16,9 +16,9 @@ component = "outbound-http-to-same-app"
 
 [component.tinygo-hello]
 source = "tinygo-hello/main.wasm"
-allowed_http_hosts = [
-    "https://random-data-api.fermyon.app",
-    "https://postman-echo.com",
+allowed_outbound_hosts = [
+    "https://random-data-api.fermyon.app:443",
```

Comment:
I liked the ergonomics of not having to specify the default port.

Reply:
We could make `https://random-data-api.fermyon.app` an alias for `https://random-data-api.fermyon.app:443` (i.e., if the port is not specified than the default port for the scheme is assumed).

Reply:
This has been done (see [this comment](https://github.com/fermyon/spin/pull/1988#issuecomment-1786949253))